core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -559,6 +560,10 @@ public class GeyserSession implements GeyserConnection, GeyserCommandSource {
 
     private MinecraftProtocol protocol;
 
+    private float tickRate = 20.0f;
+
+    private boolean frozen = false;
```

Comment:
```suggestion
    private boolean tickingFrozen = false;
```
also it seems like the `tickRate` field is unecessary

Reply:
I think tickrate should stay for the future, in case any timing related stuff needs to be done, you can use the sessions current tick rate

Reply:
then it can be added by what needs it in the future

Reply:
Fixed in latest commit.